WordPress/Classes/Models/ReaderSearchTopic.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+@objc public class ReaderSearchTopic : ReaderAbstractTopic
+{
+    override public class var TopicType: String {
```

Comment:
Nipticky: Since `class var` ~ `static var` (former one can be overriden, second one cannot), and we already've got few `static var`'s in lowercase, i'm having second thoughts as per whether we wanna capitalize this or nope.